test/python/test_jacobian.py

```diff
@@ -295,7 +368,7 @@ def test_net_rop_ddP(self):
         drop_num = self.rop_ddP(mode="net")
         self.assertNear(drop[self.rxn_idx], drop_num, self.rtol)
 
-    def rate_ddX(self, spc_ix, mode=None, const_t=True, rtol_deltac=1e-6, atol_deltac=1e-20):
+    def rate_ddX(self, spc_ix, mode=None, const_t=True, rtol_deltac=1e-6, atol_deltac=1e-20, ddX=True):
```

Comment:
Please wrap this to less than 88 characters (I realize the existing line was already over, but this just makes it worse).